formats/schemas/january-2015.txt

```diff
@@ -0,0 +1,166 @@
+Changes For January 2015
+========================
+
+
+The list of the key changes for the January 2015 major release of the
+ome-xml data model. This schema release will tie in with the Bio-Formats
+5.1 release.
+
+The new major release of the schema has a new namespace and all version
+numbers are reset to 1. As a major release, any file that validated
+correctly using the last major release will probably not validate
+correctly using this new release. Some files that failed to validate
+before will now be valid. It is important to update any file readers and
+writers to understand the changes.
+
+The version number of all schema files is now 1.
+
+This schema uses the new namespace:
+
+::
+
+    http://www.openmicroscopy.org/Schemas/[NameSpaceTitle]/2015-01/
+
+For the OME schema
+
+::
+
+    http://www.openmicroscopy.org/Schemas/OME/2015-01/
+
+and the schema file is located at
+
+::
+
+    http://www.openmicroscopy.org/Schemas/OME/2015-01/ome.xsd
+
+Overview of changes
+-------------------
+
+- This release is preparation for the release of Bio-Formats 5.1 and OMERO
```

Comment:
Yes, but this branch in this git repository *is* the 5.1 documentation.  So mentioning this here is not important.